internal/validation/v2alpha1/rules.go

```diff
@@ -56,6 +58,24 @@ func validateRules(ctx context.Context, client client.Client, parentAttributePat
 	return problems
 }
 
+func validatePath(validationPath string, rulePath string) (problems []validation.Failure) {
+	problems = append(problems, validateEnvoyTemplate(validationPath+".path", rulePath)...)
+	return problems
+}
+
+func validateEnvoyTemplate(s string, path string) []validation.Failure {
+	segments := strings.Split(strings.TrimLeft(path, "/"), "/")
```

Comment:
You can firstly check if you path has over 1 `{**}`. If it does, you can fail early.
```go
if strings.Count("{**}", path) > 1 {
    return []validation.Failure{}
}
```